Anarchist Library.js

```diff
@@ -0,0 +1,452 @@
+{
+	"translatorID": "1a31e4c5-22ed-4b5b-a75f-55476db29a44",
+	"label": "Anarchist Library",
+	"creator": "Sister Baæ'l",
+	"target": "https://theanarchistlibrary\\.org/(latest|library|stats/popular|category/topic|category/author|special/index|search)",
+	"minVersion": "7.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2025-10-07 11:18:38"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2025 Dandelion Good and the righteous Anti Toil Theologians at Iliff
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero.  If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+/*
+***** BEGIN ATTRIBUTION BLOCK *****
+This translator was developed by Dandelion Good.
+
+If you do any work on this translator, please add yourself here <3.
+*/
+
+var allAttachmentTypes = {
+	"Plain PDF": { ext: ".pdf", mimeType: "application/pdf" },
+	"A4 PDF": { ext: ".a4.pdf", mimeType: "application/pdf" },
+	"Letter PDF": { ext: ".lt.pdf", mimeType: "application/pdf" },
+	EPub: { ext: ".eupb", mimeType: "application/epub+zip" },
+	"Printer-friendly HTML": { ext: ".html", mimeType: "text/html" },
+	LaTeX: { ext: ".tex", mimeType: "application/x-tex" },
+	"Plain Text": { ext: ".muse", mimeType: "text/plain" },
+	"Source Zip:": { ext: ".zip", mimeType: "application/zip" },
+	Snapshot: { ext: "snapshot", mimeType: "text/html" }
+};
+
+function getSearchResults(doc, checkOnly) {
+	let items = {};
+	let results = doc.querySelectorAll('a.list-group-item');
+	for (let i = 0; i < results.length; i++) {
+		if (checkOnly) {
+			return true;
+		}
+		items[results[i].href] = ZU.trimInternal(text(results[i], "strong"));
+	}
+	return items;
```

Comment:
Let's just align this fully with the standard format:

```suggestion
	var items = {};
	var found = false;
	var rows = doc.querySelectorAll("a.list-group-item");
	for (let row of rows) {
		let href = row.href;
		let title = ZU.trimInternal(text(row, "strong"));
		if (!href || !title) continue;
		if (checkOnly) return true;
		found = true;
		items[href] = title;
	}
	return found ? items : false;
```